src/version.rs

```diff
@@ -480,7 +496,6 @@ mod test {
     #[test]
     fn test_nevra_parse_edge_cases() {
         assert_eq!(Nevra::parse_values("foo"), ("foo", "", "", "", ""));
-        assert_eq!(Nevra::parse_values("foo-1.2"), ("foo", "", "1.2", "", ""));
```

Comment:
This is obviously a malformed NEVRA, and while this felt like the best way to parse it given the circumstances, I couldn't figure out a scheme that would maintain this while having the correct behavior with the name.

And I'm much happier to have a parser that works correctly for well-formed NEVRA, than to try to parse broken NEVRAs in a way that "feels reasonable" but is not actually any kind of specified way.